File: Sway3i/src/main/java/com/sway3i/dto/TeacherDemand/Response/TeacherDemandResponseDTO.java
package com.sway3i.dto.TeacherDemand.Response;

import com.sway3i.entities.User;
import com.sway3i.entities.enums.DemandStatus;
import com.sway3i.entities.enums.EducationLevel;
import lombok.*;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class TeacherDemandResponseDTO {
    private Long id;
    private User createdBy;
    private String subject;
    private EducationLevel educationLevel;
    private String description;
    private DemandStatus status;

}

File: Sway3i-FrontEnd/src/app/dto/TeacherDemand/responses/teacher-demand-response-DTO.ts
import { User } from "src/app/modal/entities/user";
import { DemandStatus } from "src/app/modal/enums/demand-status-enum";
import { EducationLevel } from "src/app/modal/enums/education-level-enum";

export interface TeacherDemandResponseDTO {
  id: number;
  createdBy: User;
  subject: string;
  educationLevel: EducationLevel;
  description: string;
  status: DemandStatus;
}
